app/index.js

```diff
@@ -87,6 +87,13 @@ module.exports = class extends Generator {
 					message: 'A short description of your project:'
 				},
 				{
+					type: 'confirm',
+					name: 'useHandlebars',
+					message: 'Do you want to use Handlebars in your project?',
+					default: true, // TODO See L44
+					store: true
+				},
+				{
```

Comment:
What do you think about using a `list` prompt instead of a confirm?

```javascript
{
    type: 'list',
    name: 'projectType',
    message: 'What to you want to build?',
    choices: [
      'A static website',
      'A single page application'
    ],
    store: true
  }
```

This option should be described in the README as:

```
| `projectType` | Wether to use Baumeister as static site generator using Handlebars and Frontmatters or using developing a single page app using plain HTML and Framework specific HTML templates |
```